sdk/authorization/arm-authorization/test/sampleTest.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * Code generated by Microsoft (R) AutoRest Code Generator.
+ * Changes may cause incorrect behavior and will be lost if the code is regenerated.
+ */
+
+import {
+  Recorder,
+  RecorderStartOptions,
+  env
+} from "@azure-tools/test-recorder";
+import { assert } from "chai";
+import { Context } from "mocha";
+
```

Comment:
@kazrael2119 The ci failed. Could you take a look?

https://dev.azure.com/azure-sdk/public/_build/results?buildId=2161222&view=logs&j=b90ae18a-ab13-53d3-1df4-bcba2aec1a70&t=ac4fb594-9cc2-50fc-bf40-ff99f9e7d5ba